lib/mongo/server_selector/nearest.rb

```diff
@@ -41,13 +41,13 @@ def name
         :nearest
       end
 
-      # Whether the slaveOk bit should be set on wire protocol messages.
+      # Whether the secondaryOk bit should be set on wire protocol messages.
       #   I.e. whether the operation can be performed on a secondary server.
       #
       # @return [ true ] true
       #
       # @since 2.0.0
-      def slave_ok?
+      def secondary_ok?
```

Comment:
This looks like a public method, in which case I believe the `slave_ok?` method should be added back as an alias of `secondary_ok?` with a @deprecated note.

Reply:
And a ticket to remove these under 3.0 epic.

Reply:
And then the tests should test both methods.

Reply:
Good point, I'll check other public methods as well.

Reply:
Added aliases and tests.